packages/opentelemetry-sdk-trace-base/src/export/SpanExporter.ts

```diff
@@ -28,10 +28,15 @@ export interface SpanExporter {
   /**
    * Called to export sampled {@link ReadableSpan}s.
    * @param spans the list of sampled Spans to be exported.
+   * @param resultCallback
+   * @param exportTimeoutMillis optional - currently only used by BatchSpanProcessorBase
+   * @param onError optional - currently only used by BatchSpanProcessorBase
    */
   export(
     spans: ReadableSpan[],
-    resultCallback: (result: ExportResult) => void
+    resultCallback: (result: ExportResult) => void,
+    exportTimeoutMillis?: number,
```

Comment:
We can't add optional parameter in the middle of the arguments because that would be a breaking change

Reply:
That makes sense. Thanks for your feedback.